docs/site/Server.md

```diff
@@ -139,6 +139,15 @@ const app = new RestApplication({
 });
 ```
 
+### Use a self-hosted API Explorer
+
+Hosting the API Explorer at an external URL has few downsides, for example a
+working internet connection is required to explore the API. As a recommended
+alternative, LoopBack comes with an extension that provides a self-hosted
+Explorer UI. Please refer to
+[Self-hosted REST API Explorer](./Self-hosted-rest-api-explorer.md) for more
```

Comment:
The file name `./Self-hosted-rest-api-explorer.md` is different from `docs/site/Self-hosted-REST-API-Explorer.md` by case.

Reply:
Good catch 👍